library/ajax/pharmacy_autocomplete/search.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
As discussed above, do not embed assets (jquery and jquery-ui) and instead use ones already in public/assets/